__tests__/integration/one-app.spec.js

```diff
@@ -1668,3 +1668,213 @@ describe('heapdump', () => {
     expect(size).toBeLessThanOrEqual(100 * oneMegabyte);
   });
 });
+
+describe('Routes sanity checks', () => {
+  const defaultFetchOptions = createFetchOptions();
+  let originalModuleMap;
+  const oneAppLocalPortToUse = getRandomPortNumber();
+  const oneAppMetricsLocalPortToUse = getRandomPortNumber();
+  const fetchUrl = `https://localhost:${oneAppLocalPortToUse}`;
+
+  let browser;
+
+  beforeAll(async () => {
+    originalModuleMap = readModuleMap();
+    ({ browser } = await setUpTestRunner({ oneAppLocalPortToUse, oneAppMetricsLocalPortToUse }));
+  });
+
+  afterAll(async () => {
+    await tearDownTestRunner({ browser });
+    writeModuleMap(originalModuleMap);
+  });
+
+  test('Request: /_/status', async () => {
+    const response = await fetch(
+      `${fetchUrl}/_/status`,
+      {
+        ...defaultFetchOptions,
+        method: 'GET',
+        headers: {
+          origin: 'test.example.com',
+        },
+      }
+    );
+    const rawHeaders = response.headers.raw();
+
+    expect(response.status).toBe(200);
+    expect(Object.keys(rawHeaders)).toEqual([
+      'x-frame-options',
+      'x-content-type-options',
+      'strict-transport-security',
+      'x-xss-protection',
+      'referrer-policy',
+      'one-app-version',
+      'content-type',
+      'content-length',
+      'etag',
+      'vary',
+      'date',
+      'connection',
+    ]);
+    expect(rawHeaders.connection).toEqual(['close']);
+    expect(rawHeaders['content-length']).toEqual(['2']);
+    expect(rawHeaders['content-type']).toEqual(['text/plain; charset=utf-8']);
+    expect(rawHeaders['referrer-policy']).toEqual(['same-origin']);
+    expect(rawHeaders['strict-transport-security']).toEqual([
+      'max-age=15552000; includeSubDomains',
+    ]);
+    expect(rawHeaders.vary).toEqual(['Accept-Encoding']);
+    expect(rawHeaders['x-content-type-options']).toEqual(['nosniff']);
+    expect(rawHeaders['x-frame-options']).toEqual(['DENY']);
+    expect(rawHeaders['x-xss-protection']).toEqual(['1; mode=block']);
```

Comment:
What about an inline snapshot of the headers?

Reply:
we could use inline snapshot, only problem is that some header values are dynamic, like `date` and `etag`, maybe there's a way to mock them

Reply:
check out asymmetric snapshots